### ISSUE TITLE
Reporting results of statistical tests features

### ISSUE THREAD

**Issue by dimleyk** [jasp-stats/jasp-desktop#996]
Is it possible to add a reporting feature which will convert the statistical tables into words and meanings.
For instance, take the example shown here about ANCOVA : http://www.statisticshell.com/docs/ancova.pdf
You can take the reporting sections out of this file, and create a template that will use specific fields in JASP analysis and embed them in the textual report. If for instance, you request to add an effect size, this effect size will also appear in the textual report of the statistical test.
I THINK IT WILL BE GREAT!

**Comment by jonathon-love** [comment]
yes, i think this is a great idea, and i've been wanting to do something like it for a while... stay tuned (though it probably won't happen very soon).

**Comment by dostodabsi** [comment]
I think Daniel Lakens has done something [similar](https://github.com/Lakens/Perfect-t-test).

**Comment by dimleyk** [comment]
:+1: @dostodabsi - i saw the youtube of Lakens' work - very impressive, though the report is too way comprehensive, and actually the most important reports are summarized in few lines. The plots and the tables are given by JASP - and the only thing left is the textual report :-)

**Comment by dimleyk** [comment]
If you could integrate the reporting like this : http://info.statisticssolutions.com/software-free-trial  this will be heaven!
